cmake/leatherman.cmake.in

```diff
@@ -111,7 +116,7 @@ macro(enable_cppcheck)
 endmacro()
 
 # We set this here so that enable_cpplint() can find it
-set(LEATHERMAN_CPPLINT_PATH "${CMAKE_CURRENT_LIST_DIR}/../scripts/cpplint.py")
+set(LEATHERMAN_CPPLINT_PATH "${CMAKE_CURRENT_LIST_DIR}/../../scripts/cpplint.py")
```

Comment:
This seems like it'd be similar to treatment of `LEATHERMAN_CMAKE_DIR`. Why isn't it the same problem?